### PR TITLE
Wenbo-New Pull 2

### PR DESCRIPTION
**Pull request by alanmin2015**

### CODE REVIEW

**Review by Sonnto — APPROVED** [review]

**Comment by alanmin2015** [comment]
Wenbo

File: Models/Lab.cs [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+        public ICollection<Project> Projects { get; set; }
+    }
+
+    public class LabDto
+    {
+        public int LabId { get; set; }
+        public string LabName { get; set; }
+    }
+}
```
Comment (beartisan) [inline Models/Lab.cs:24]: lab models approved

**Review by beartisan — APPROVED** [review]

**Review by Sonnto — APPROVED** [review]
My SECOND approve. I'm just gonna approve it.

**Review by beartisan — APPROVED** [review]
Bea1

**Pull request merged**